packages/datadog-plugin-next/src/index.js

```diff
@@ -32,6 +32,22 @@ function createWrapHandleApiRequest (tracer, config) {
   }
 }
 
+function createWrapRenderToResponse (tracer, config) {
```

Comment:
i feel like all these wrap functions are super repetitive, but not gonna block on that.

Reply:
They vary slightly, just enough to make it difficult to reuse the same code.